Makefile

```diff
@@ -1,38 +1,22 @@
 CWD=$(shell pwd)
-PYVER ?= $(shell for p in python3 python2; do \
-	out=$$(command -v $$p 2>&1) && echo $$p && exit; done; exit 1)
 
 YAML_FILES=$(shell find cloudinit tests tools -name "*.yaml" -type f )
 YAML_FILES+=$(shell find doc/examples -name "cloud-config*.txt" -type f )
 
-PIP_INSTALL := pip install
-
-ifeq ($(PYVER),python3)
-  pyflakes = pyflakes3
-  unittests = unittest3
-  yaml = yaml
-else
-ifeq ($(PYVER),python2)
-  pyflakes = pyflakes
-  unittests = unittest
-else
-  pyflakes = pyflakes pyflakes3
-  unittests = unittest unittest3
-endif
-endif
+PYTHON = python3
+PIP_INSTALL := pip3 install
 
 ifeq ($(distro),)
   distro = redhat
 endif
 
-READ_VERSION=$(shell $(PYVER) $(CWD)/tools/read-version || \
-  echo read-version-failed)
-CODE_VERSION=$(shell $(PYVER) -c "from cloudinit import version; print(version.version_string())")
+READ_VERSION=$(shell $(PYTHON) $(CWD)/tools/read-version || echo read-version-failed)
+CODE_VERSION=$(shell $(PYTHON) -c "from cloudinit import version; print(version.version_string())")
 
 
 all: check
 
-check: check_version test $(yaml)
+check: check_version test yaml
```

Comment:
we did skip `yaml` on py2 (that's why it's in a variable), but as we support py3 there is no need for a variable